comp/snmptraps/bundle_test.go

```diff
@@ -0,0 +1,56 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2023-present Datadog, Inc.
+
+package snmptraps
+
+import (
+	"testing"
+
+	"github.com/DataDog/datadog-agent/comp/core/config"
+	"github.com/DataDog/datadog-agent/comp/core/hostname/hostnameimpl"
+	"github.com/DataDog/datadog-agent/comp/core/log"
+	"github.com/DataDog/datadog-agent/comp/ndmtmp"
+	trapsconfig "github.com/DataDog/datadog-agent/comp/snmptraps/config"
+	"github.com/DataDog/datadog-agent/comp/snmptraps/formatter"
+	"github.com/DataDog/datadog-agent/comp/snmptraps/forwarder"
+	"github.com/DataDog/datadog-agent/comp/snmptraps/listener"
+	"github.com/DataDog/datadog-agent/comp/snmptraps/oidresolver"
+	trapsserver "github.com/DataDog/datadog-agent/comp/snmptraps/server"
+	"github.com/DataDog/datadog-agent/comp/snmptraps/status"
+	"github.com/DataDog/datadog-agent/pkg/aggregator/mocksender"
+	"github.com/DataDog/datadog-agent/pkg/aggregator/sender"
+	"github.com/stretchr/testify/require"
+	"go.uber.org/fx"
+)
+
+type deps struct {
+	fx.In
+	Config    trapsconfig.Component
+	Formatter formatter.Component
+	Forwarder forwarder.Component
+	Listener  listener.Component
+	Resolver  oidresolver.Component
+	Server    trapsserver.Component
+	Status    status.Component
+}
+
+func TestBundleDependencies(t *testing.T) {
+	require.NoError(t, fx.ValidateApp(
```

Comment:
You can simplify the test by using [fxutil.TestBundle](https://github.com/DataDog/datadog-agent/blob/e6db9b4c9be9f058f46183f6667faf3852c8655a/comp/aggregator/bundle_test.go#L19).

Reply:
Done.